app/assets/javascripts/map.js

```diff
@@ -154,21 +167,33 @@ function loadMarker(markerObject){
     map: map
   });
 
-  addInfoWindow(markerObject, marker);
+  new InfoBox(attraction, marker)
 }
 
-function addInfoWindow(markerObject, marker){
-  var contentString = "<div>" +
-    '<p>' +
-    markerObject.name +
-    '</p>' +
-    '</div>';
-
-    var infoWindow = new google.maps.InfoWindow({
-      content: contentString
-    });
+//-----------InfoBox----------------
+var InfoBox = function(attraction, marker){
+  this.contentString ='<div>' +
```

Comment:
also consider that `[].join('')` can join segments together.  That's a nice "cheap templating" solution instead of having to track `+`.